src/app/public/modules/vertical-tabset/fixtures/vertical-tabset-empty-group.component.fixture.html

```diff
@@ -1,4 +1,6 @@
-<sky-vertical-tabset showTabsText='Tab list'>
+<sky-vertical-tabset
+  showTabsText='Tab list'
+  [maintainTabContent]="maintainTabContent">
```

Comment:
Closing bracket needs to be on a new line.

Reply:
Done